minichain/contract.py

```
@@ -1,169 +1,13 @@
-                    if "__" in node.value:
-                        logger.warning("Rejected string literal with double-underscore.")
-                        return False
-                if isinstance(node, ast.JoinedStr): # f-strings
-                    logger.warning("Rejected f-string usage.")
-                    return False
-            return True
-        except SyntaxError:
-            return False
+if __name__ == "__main__":
+    # Process Pool example
+    with multiprocessing.Pool(processes=4) as pool:
```

Ruff (0.15.7)

[error] Undefined name `multiprocessing`

(F821)